### PR TITLE
Fixes zeroes in station time

### LINKED ISSUE #36417: STATION TIME IS BROKEN
(WEB REPORT BY: angryturnip REMOTE: 172.93.101.77:7777)
# Revision
d4972dc7f07a152012893dab931d6b5c5ddc1d74

# Description
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
# Steps to Reproduce
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
# What you Expected
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
# What Actually Happened
Station Time: 12:3

### PR DESCRIPTION
**Pull request by DeityLink**
image

Fixes #36417

:cl:
* bugfix: Fixed zeroes in station time.

### CODE REVIEW

**Comment by west3436** [comment]
Fix the spacing please

**Comment by SonixApache** [comment]
closes #36417

remove the spaces

File: code/__HELPERS/time.dm [hunk 1]
```diff
@@ -38,9 +38,8 @@
 /proc/worldtime2text(timestamp = world.time, give_seconds = FALSE)
 	if(timestamp == world.time)
 		timestamp -= Master.time_taken_to_init
-	return "[(round(((timestamp / 600) + 55) / 60) + 11) % 24]:\
-	[(((timestamp / 600) + 55) % 60) < 10 ? add_zero(((timestamp / 600) + 55) % 60, 1) : ((timestamp / 600) + 55) % 60]\
-	[give_seconds ? ":[(timestamp / 10 % 60) < 10 ? add_zero(timestamp / 10 % 60, 1) : timestamp / 10 % 60]" : ""]"
+	return "[add_zero((round(((timestamp / 600) + 55) / 60) + 11) % 24, 2)] : [add_zero(round(((timestamp / 600) + 55) % 60), 2)]\
```
Comment (DeityLink) [inline code/__HELPERS/time.dm]:
```suggestion
	return "[add_zero((round(((timestamp / 600) + 55) / 60) + 11) % 24, 2)]:[add_zero(round(((timestamp / 600) + 55) % 60), 2)]\
```

**Comment by DeityLink** [comment]
Alright fine, spaces gone

**Pull request merged**